libraries/AP_Logger/AP_Logger_File.h

```diff
@@ -17,9 +17,13 @@
 #if AP_FILESYSTEM_LITTLEFS_ENABLED
 #define HAL_LOGGER_WRITE_CHUNK_SIZE 2048
 #else
+#if defined(STM32H7)
```

Comment:
It would be nice to keep this in `chibios.h` rather than spreading HAL-specific defines in here